test/analysis_ee_Z_bb_FCCSW_cfg.py

```diff
@@ -0,0 +1,129 @@
+'''Example configuration file for an ee->ZH->mumubb analysis in heppy, with the FCC-ee
+
```

Comment:
I fear there is a lot of duplication here w/r to the original file. 
if we change the original in the future, we will have to propagate the flag. 
maybe set a variable in the original cfg like : run_papas
and if False, one goes to reading mode? 
Need to see how this plays with the unittests

Reply:
I have set a variable as you suggested and have created two unit_tests so that they can each have a different configuration. I needed each unit test to be able to say which config it wanted, so (and you may not like this) I put the variable into the configuration.py file.  I am happy to change this if you have a better plan
class PapasRuntype(object):
    '''Describes the run parameters:
    '''
    from_fccsw = False